extensions/amp-subscriptions/0.1/test/test-platform-store.js

```diff
@@ -398,6 +398,55 @@ describes.realWin('Platform store', {}, () => {
     });
   });
 
+  describe('selectPlatformForLogin', () => {
+    let localPlatform, localFactors;
+    let anotherPlatform, anotherFactors;
+
+    beforeEach(() => {
+      localFactors = {};
+      localPlatform = new SubscriptionPlatform();
+      sandbox.stub(localPlatform, 'getServiceId').callsFake(() => 'local');
+      sandbox.stub(localPlatform, 'getBaseScore')
+          .callsFake(() => 10000);  // base score does not matter.
+      sandbox.stub(localPlatform, 'getSupportedScoreFactor')
+          .callsFake(factor => localFactors[factor]);
+      anotherFactors = {};
+      anotherPlatform = new SubscriptionPlatform();
+      sandbox.stub(anotherPlatform, 'getServiceId').callsFake(() => 'another');
+      sandbox.stub(anotherPlatform, 'getBaseScore')
+          .callsFake(() => 0);
+      sandbox.stub(anotherPlatform, 'getSupportedScoreFactor')
+          .callsFake(factor => anotherFactors[factor]);
+      platformStore.resolvePlatform('local', localPlatform);
```

Comment:
Suggest you swap the order of `local` and `anothertPlatform`.  Otherwise local wins a tie because of default order and not because of the explicit test.